docs/getting-started/quick-start.md

```diff
@@ -72,15 +67,107 @@ import { FormatPainter, SlashCommand } from 'ckeditor5-premium-features';
 
 import 'ckeditor5/dist/styles.css';
 
-ClassicEditor
+await ClassicEditor
 	.create( document.querySelector( '#editor' ), {
 		plugins: [ Essentials, FormatPainter, SlashCommand ],
 		toolbar: [ 'formatPainter' ],
 		licenseKey: 'your-license-key'
 	} )
-	.catch( err => {
-		console.error( err );
-	} );
+```
+
+## Using CKEditor&nbsp;5 from CDN
+
+CDN is an effortless method of running CKEditor 5. You can start using it in just a few steps and with a few tags.
```

Comment:
I'd be cautious about using the word `effortless`, as - depending on the setup - it may require more work than the NPM setup. I'd suggest replacing `effortless` with `alternative`.

```suggestion
CDN is an alternative method of running CKEditor 5. You can start using it in just a few steps.
```

Reply:
Yeah, you're right. That was marketing jabber foremost. Will do.